### ISSUE TITLE
Feature request: langID in multilingual pipelines

### ISSUE THREAD

**Issue by loretoparisi** [nlp-uoregon/trankit#1]
Thanks for this framework! 
It could be worth to add a language identification task to avoid using `p.set_active( lang )`. For langId a very robust, fast and tiny one could be [FastText](https://fasttext.cc/blog/2017/10/02/blog-post.html) or a BERT model (better integration, but computational intensive) So the multilingual pipeline would become:

```python
from trankit import Pipeline
p = Pipeline(lang='auto', gpu=True, cache_dir='./cache') # auto means language identification active
p.tokenize('Rich was here before the scheduled time.')
p.ner('وكان كنعان قبل ذلك رئيس جهاز الامن والاستطلاع للقوات السورية العاملة في لبنان.')
```

**Comment by minhvannguyen** [comment]
> Thanks for this framework!
> It could be worth to add a language identification task to avoid using `p.set_active( lang )`. For langId a very robust, fast and tiny one could be [FastText](https://fasttext.cc/blog/2017/10/02/blog-post.html) or a BERT model (better integration, but computational intensive) So the multilingual pipeline would become:
> 
> ```python
> from trankit import Pipeline
> p = Pipeline(lang='auto', gpu=True, cache_dir='./cache') # auto means language identification active
> p.tokenize('Rich was here before the scheduled time.')
> p.ner('وكان كنعان قبل ذلك رئيس جهاز الامن والاستطلاع للقوات السورية العاملة في لبنان.')
> ```

Thank you for your suggestion. This is a useful feature and we have it in our development plan. We will update Trankit with this feature soon. Thank you!

**Comment by minhhdvn** [comment]
Hi @loretoparisi,
We're happy to let you know that your suggestion has been implemented in the new version v1.0.0 of Trankit. Please check it out [here](https://trankit.readthedocs.io/en/latest/news.html#auto-mode-for-multilingual-pipelines).
Thank you.

**Comment by loretoparisi** [comment]
@minhhdvn that's super!!! love the name Auto Mode! I will try it out definitevely!
